core/src/components/accordion/accordion.ts

```diff
@@ -294,28 +296,29 @@ export interface AccordionItemCommonPropsAndState {
 	slotItemBody: SlotContent<AccordionItemContext>;
 	/**
 	 * Structure of the accordion-item. The default item structure is: accordion-item
-	 * contains accordion header and accordion collapse; the accordion header contains the accordion button
-	 * (that contains `slotItemHeader`), while the accordion collapse contains the accordion body (that contains slotItemBody).
+	 * contains accordion header and accordion-item body container; the accordion header contains the accordion button
+	 * (that contains `slotItemHeader`), while the accordion-item body container contains the accordion body (that contains `slotItemBody`).
+	 * The itemTransition it applied on this element.
 	 */
 	slotItemStructure: SlotContent<AccordionItemContext>;
 	/**
-	 * Classes to add on the accordion-item DOM element.
+	 * CSS classes to add on the accordion-item DOM element.
 	 */
 	itemClass: string;
 	/**
-	 * Classes to add on the accordion-item header DOM element.
+	 * CSS classes to add on the accordion-item header DOM element.
 	 */
 	itemHeaderClass: string;
 	/**
-	 * Classes to add on the accordion-item collapse DOM element.
+	 * CSS classes to add on the accordion-item collapse DOM element.
 	 */
 	itemButtonClass: string;
 	/**
-	 * Classes to add on the accordion-item collapse DOM element.
+	 * CSS classes to add on the accordion-item collapse DOM element.
```

Comment:
I think you should also update this description, which is the same as the previous one, which should also be updated.

Reply:
I've updated the description for both (Accordion and Accordion Item), thx for the remark